deploy/olm-catalog/integreatly-operator/2.6.1/integreatly-operator.v2.6.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,465 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "integreatly.org/v1alpha1",
+          "kind": "RHMI",
+          "metadata": {
+            "name": "INSTALLATION_NAME"
+          },
+          "spec": {
+            "deadMansSnitchSecret": "INSTALLATION_PREFIX-deadmanssnitch",
+            "namespacePrefix": "INSTALLATION_PREFIX-",
+            "operatorsInProductNamespace": "OPERATORS_IN_PRODUCT_NAMESPACE",
+            "pagerDutySecret": "INSTALLATION_PREFIX-pagerduty",
+            "selfSignedCerts": "SELF_SIGNED_CERTS",
+            "smtpSecret": "INSTALLATION_PREFIX-smtp",
+            "type": "INSTALLATION_TYPE",
+            "useClusterStorage": "USE_CLUSTER_STORAGE"
+          }
+        },
+        {
+          "apiVersion": "integreatly.org/v1alpha1",
+          "kind": "RHMI",
+          "metadata": {
+            "name": "example-rhmi"
+          },
+          "spec": {
+            "deadMansSnitchSecret": "redhat-rhmi-deadmanssnitch",
+            "namespacePrefix": "redhat-rhmi-",
+            "pagerDutySecret": "redhat-rhmi-pagerduty",
+            "selfSignedCerts": true,
+            "smtpSecret": "redhat-rhmi-smtp",
+            "type": "managed",
+            "useClusterStorage": "true"
+          }
+        }
+      ]
+    capabilities: Basic Install
+    categories: RHMI integration
+    certified: "false"
+    containerImage: quay.io/integreatly/integreatly-operator:v2.6.1-rc3
```

Comment:
Not sure if it's ok to still reference an RC image here? Testing for v2.6.1 commenced but was then halted when the decision to skip v2.6 was made. As a result we don't have an official `integreatly-operator:v2.6.1` image to reference

Reply:
I think this is ok. This will only have an impact if you run the operator via OLM, and it doesn't impact production environments at all.